src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -50,6 +51,13 @@ export class ProsemirrorAdapter {
     @Event()
     private change: EventEmitter<string>;
 
+    @Watch('value')
+    public watchValue(newValue: string) {
+        if (newValue === '') {
+            this.view.dom.innerHTML = '';
```

Comment:
I think we might want to leave a note here, for our future selves 😊

```suggestion
            // Note!
            // This is only safe because we are setting `innerHTML` to an empty
            // string. If refactoring, please make sure this isn't accidentally
            // turned into something like `this.view.dom.innerHTML = newValue`.
            this.view.dom.innerHTML = '';
```

`fixup! feat(text-editor): add watcher for value prop`